src/libAtomVM/opcodesswitch.h

```diff
@@ -2213,7 +2213,16 @@ HOT_FUNC int scheduler_entry_point(GlobalContext *glb)
                 #endif
 
                 #ifdef IMPL_EXECUTE_LOOP
-                    int local_process_id = term_to_local_process_id(x_regs[0]);
+                    term process_term = x_regs[0];
```

Comment:
I would call it just `recipient` or with any name that is not biased towards a process.